py3status/modules/mpris.py

```diff
@@ -0,0 +1,236 @@
+# -*- coding: utf-8 -*-
+"""
+Display information about the current song and video playing on player with
+mpris support.
+
+Configuration parameters:
+    color_none: text color when no player is running, defaults to color_bad
+    color_paused: text color when song is paused, defaults to color_degraded
+    color_playing: text color when song is playing, defaults to color_good
+    color_stopped: text color when song is stopped, defaults to color_bad
+    format: see placeholders below
+    format_video: see placeholders below, keep in mind that {artist} and {album}
+            are empty
+    format_error: define output if one or more informations couldn't read,
+            keep in mind that some playceholders may be work
+    format_none: define output if no player is running
+    state_paused: text for placeholder {state} when song is paused
+    state_playing: text for placeholder {state} when song is playing
+    state_stopped: text for placeholder {state} when song is stopped
+    player_priority: priority of the players.
+            Keep in mind that the state has a higher priority than
+            player_priority. So when player_priority is "mpd,bomi" and mpd is
+            paused and bomi is playing than bomi wins.
+
+Format of status string placeholders:
+    {state} playback status of the player
+    {album} album name
+    {artist} artiste name (first one)
+    {time} time duration of the song
+    {title} name of the song
+
+i3status.conf example:
+
+```
+mpris {
+    format = "{state} {artist} - {title} -> {time}"
+    format_video = "{state} {title} -> {time}"
+    format_none = "no player"
+    format_error = "{state} Unknown"
+    player_priority = "mpd,cantata,vlc,bomi"
+}
+```
+
+Requires:
+    dbus
+
+@author Pierre Guilbert, Jimmy Garpehäll, sondrele, Andrwe, Moritz Lüdecke
+"""
+
+from datetime import timedelta
+from time import time, sleep
+import re
+import dbus
```

Comment:
pydbus would be better to use 
https://www.freedesktop.org/wiki/Software/DBusBindings/